include/glow/IR/IRUtils.h

```diff
@@ -123,13 +123,53 @@ class ValueUses {
   const_iterator end() const { return const_iterator{v_, 1}; }
 };
 
+/// A convenience iterator type that returns element pointers instead of element
```

Comment:
I am okay with adding this as a temporary solution, but I think that we should remove this in the next commit and just change all uses.

Reply:
After a discussion with others, let's just use the standard ilist iterators returning references and update all the loop bodies.

Reply:
Fixed.